examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -551,10 +681,33 @@ def add_images(client, customer_id, campaign_resource_name):
         help="Whether or not the campaign will use image extensions.",
     )
 
+    # The name of the customizer attribute to be used in the ad customizer, which
```

Comment:
Change "to be used" to "used". Docstrings and design docs should be in the present tense.

Reply:
done